src/pages/Contact/Contact.css

```diff
@@ -1,32 +1,18 @@
-@media all and (min-width: 480px) {
-    .background {
-        background-image: url('../../pics/adult-backlit-beach.jpg');
-        background-position: center;
-        background-repeat: no-repeat;
-        background-size: cover;
-        height: 100%;
-    }
-    .Contact {
-        margin: 0 auto;
-        max-width: 320px;
-        background-color: white;
-    }
-
-    .Contact form {
-        width: 100%;
-    }
+input[type=text] {
+    width: 100% !important;
```

Comment:
You don't need to use !important tag here.

Reply:
removed.